cmd/swarm/swarm-smoke/main.go

```diff
@@ -122,6 +124,18 @@ func main() {
 			Usage:       "whether to fetch content from a single node or from all nodes",
 			Destination: &single,
 		},
+		cli.IntFlag{
+			Name:        "nodes",
+			Value:       20,
+			Usage:       "how many nodes",
+			Destination: &nodes,
+		},
+		cli.IntFlag{
+			Name:        "store",
```

Comment:
Not sure why we need this. We specify the `filesize` for the content to upload. Why do we need this flag, can't we use the `filesize`?

Reply:
also @nonsense, we don't specify the filesize to upload in this case. that value is so far ignored since i wanted the test to make sure that we don't upload files which are for example bigger than the store size. the filesize is determined on the fly according to store size.